packages/components/Checkbox/src/Checkbox.types.ts

```diff
@@ -7,7 +7,7 @@ import { SvgProps } from 'react-native-svg';
 
 export const checkboxName = 'Checkbox';
 export type CheckboxSize = 'medium' | 'large';
-export type CheckboxShape = 'circular' | 'square';
+export type CheckboxShape = 'circular' | 'square'; // Circular is not supported on Android
```

Comment:
This seems redundant given it's already marked so for "shape" prop.

Reply:
Removed.